problems/2.functions.js

```diff
@@ -1,41 +1,57 @@
 /** Logs a custom greeting to the console in the form "Hello Avery!"
-* @param {string} name - The name of the user to greet
-* @returns {null}
-*/
+ * @param {string} name - The name of the user to greet
+ * @returns {null}
+ */
 function logGreeting() {
+    let logGreeting = "Hello Ben!"
+    console.log(logGreeting);
+};
 
-}
 
 /** Returns a custom greeting string in the form "Hello Avery!"
-* @param {string} name - The name of the user to greet
-* @returns {string} - The custom greeting
-*/
-function greeting() {
+ * @param {string} name - The name of the user to greet
+ * @returns {string} - The custom greeting
+ */
+function greeting(name = "Avery") {
+    return `Hello ${name}!`
 
 }
 
 /** Adds a number to a global variable called globalCount
  * Access globalCount like this: global.globalCount
-* @param  {number} val - The number to add to globalCount
-* @returns {null}
-* @modifies {global.globalCount}
-*/
+ * @param  {number} val - The number to add to globalCount
+ * @returns {null}
+ * @modifies {global.globalCount}
+ */
 function incrementGlobalCount() {
+    let i = 0;
```

Comment:
To update a variable you need to do reassignment and thus need the assignment operator (=). 
global.globalCount += val